File: Cpp/1304_Find_N_Unique_Integers_Sum_up_to_Zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> result(n, 0);
        
        for(int i=0; i<n/2; ++i){
            result[i*2] = i+1;
            result[i*2+1] = -i-1;
        }
        
        return result;
    }
};

File: Kotlin/1304_Find_N_Unique_Integers_Sum_up_to_Zero.kt
class Solution {
    fun sumZero(n: Int): IntArray {
        val result = IntArray(n)
        
        for(i in 0 until n/2){
            result[i * 2] = i + 1
            result[i * 2 + 1] = -i - 1
        }
        
        return result
    }
}